src/Service/VendorService/TheMovieDatabase/TheMovieDatabaseVendorService.php

```diff
@@ -72,70 +143,233 @@ protected function extractData(object $jsonContent): array
+            $match = preg_match('/(\d{4})$/u', (string) $subject->{'$'}, $subjectMatches);
 
             if ($match) {
-                $matches = array_unique(array_merge($matches, $descriptionMatches));
+                $matches = array_unique(array_merge($matches, $subjectMatches));
             }
         }
 
         $upperYear = (int) date('Y') + 2;
         $confirmedMatches = [];
 
         foreach ($matches as $matchString) {
-            $match = (int) $matchString;
+            $match = intval($matchString);
 
-            if ($match > 1850 && $match < $upperYear) {
-                $confirmedMatches[] = $matchString;
+            // "Roundhay Garden Scene ... is believed to be the oldest surviving film in existence."
+            // https://en.wikipedia.org/wiki/Roundhay_Garden_Scene
+            if ($match > 1888 && $match < $upperYear) {
+                $confirmedMatches[] = $match;
             }
         }
 
-        if (1 === count($confirmedMatches)) {
-            return $confirmedMatches[0];
-        }
+        // Ensure matches are ordered chronologically
+        sort($confirmedMatches);
 
-        return null;
+        return $confirmedMatches[0] ?? null;
     }
 
     /**
-     * Extract the director from the creators.
+     * Extract the creators from the DC list creators excluding sort entries.
      *
      * @param array $creators
      *   Search array of creators
      *
-     *   The director or null
+     * @return array
      */
-    private function getDirector(array $creators): ?string
+    private function getCreators(array $creators): array
     {
-        $directors = [];
+        $filtered = [];
 
         foreach ($creators as $creator) {
-            if (isset($creator->{'@type'}?->{'$'}) && 'dkdcplus:drt' === $creator->{'@type'}?->{'$'}) {
-                if (isset($creator->{'$'})) {
-                    $directors[] = $creator->{'$'};
+            // The "oss:sort" is a duplicate entry for sorting. E.g " Koster, Henry" for "Henry Koster"
+            if (isset($creator->{'$'})) {
+                if (!isset($creator->{'@type'}?->{'$'}) || (isset($creator->{'@type'}?->{'$'}) && 'oss:sort' !== $creator->{'@type'}?->{'$'})) {
+                    $name = $this->getSanitizedCreator($creator->{'$'});
+                    $filtered[$name] = $name;
                 }
             }
         }
 
-        // @TODO: Can there be more directors?
-        if (count($directors) > 0) {
-            return $directors[0];
+        return $filtered;
+    }
+
+    /**
+     * Get the sanitized title without additional text added by the datawell.
+     *
+     * @param string $datawellTitle
+     *
+     * @return string
+     */
+    private function getSanitizedTitle(string $datawellTitle): string
+    {
+        // E.g. "Game of thrones" for "Game of thrones. Sæson 2. Disc 3, episodes 5, 6 & 7".
+        $pos = mb_stripos($datawellTitle, '. Sæson', 0, 'UTF-8');
+        if ($pos > 0) {
```

Comment:
I thinks this title is set by "staff" and this might only hit a very small sub-set of titles.

Reply:
I have seen a lot of these. Not important now because we don't handle tv-shows. But will be needed.